packages/kamino-sdk/tests/stratCreation.test.ts

```diff
@@ -134,7 +99,13 @@ describe('Kamino strategy creation SDK Tests', () => {
     console.log('setup strategy fees tx hash', txHash);
 
     // after strategy creation we have to set the reward mappings so it autocompounds
-    let updateRewardMappingIxs = kamino.getUpdateRewardsIxs(signer.publicKey, newStrategy.publicKey);
+    let updateRewardMappingIxs = await kamino.getUpdateRewardsIxs(signer.publicKey, newStrategy.publicKey);
```

Comment:
a bit too verbose here, too much boilerplate, we should abstract this away as it repeats quite a lot and adds noise around the more interesting code

Reply:
i think it is good to have it in a couple of tests where we test this, but I agree, I can add a common function to set up a strat and rewards and everything